doc/maintaining/configuration.rst

```diff
@@ -643,6 +643,20 @@ Default value: ``False`` (however the default config file template sets it to ``
 Restricts access to 'view this version' and 'changes' in the Activity Stream pages. These links provide users with the full edit history of datasets etc - what they showed in the past and the diffs between versions. If this option is set to ``False`` then only admins (e.g. whoever can edit the dataset) can see this detail. If set to ``True``, anyone can see this detail (assuming they have permission to view the dataset etc).
 
 
+.. _ckan.auth.allow_anonymous_access:
+
+ckan.auth.allow_anonymous_access
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
It seems one of the build errors is caused here at line 649. Can you remove the extra `^` so it's just 
```
ckan.auth.allow_anonymous_access
^^^^^^^^^^^^^^^^^^^^^^^
```